src/components/Modal/Modal.jsx

```diff
@@ -57,9 +57,7 @@ class Modal extends React.Component {
   }
 }
 
-Modal.defaultProps = {
-  minWidth: 'auto',
-};
+Modal.defaultProps = {};
```

Comment:
is this change going to break anything? If minWidth is null/undefined what will be passed to style?

Reply:
fixed